Aliases/ignition-msgs@6

```diff
@@ -0,0 +1 @@
+../Formula/ignition-msgs5.rb
```

Comment:
There's already a formula for `ign-msgs6`, so we don't need an alias.

https://github.com/osrf/homebrew-simulation/blob/master/Formula/ignition-msgs6.rb

Reply:
removed, thanks!